core/templates/pages/exploration-editor-page/statistics-tab/statistics-tab.component.ts

```diff
@@ -91,6 +93,7 @@ angular.module('oppia').component('statisticsTab', {
           if (expStats.numActualStarts > 0) {
             $scope.explorationHasBeenVisited = true;
           }
+          $rootScope.$applyAsync();
```

Comment:
Why? Can you explain with a comment?

Reply:
@brianrodri why as in why is this change needed?

Reply:
@brianrodri, This is similar to the `$rootScope.$apply()` we need to add after migrating a service that uses $http.